src/radio.cpp

```diff
@@ -100,18 +100,18 @@ void zmq::radio_t::xwrite_activated (pipe_t *pipe_)
 
 void zmq::radio_t::xpipe_terminated (pipe_t *pipe_)
 {
-    for (subscriptions_t::iterator it = subscriptions.begin (); it != subscriptions.end (); ) {
```

Comment:
You are very welcome to contribute, and we are thankful for that.
Sorry to be strict, but we have many users on legacy platforms, and everytime something breaks I have to scramble and spend time to fix it up again. So if I see something that I know will be broken because we've already seen it, I can't accept it.